plugin.json

```
@@ -59,10 +59,10 @@
         "help_text": "Application ID from the Azure Active Directory. Only required for Skype for Business Online. Use this URL to configure the plugin in Azure Active Directory: ```https://SITEURL/plugins/skype4business/api/v1/auth_redirect```"
       },
       {
-        "key": "LogClientRequests",
+        "key": "ShouldLogClientRequests",
         "display_name": "Enable logging requests to S4B server",
         "type": "bool",
-        "help_text": "Logging requests sent to S4B server for debugging purposes. Used only with Skype for Business Server."
+        "help_text": "Logging requests sent to S4B server for debugging purposes. Used only with Skype for Business Server. These logs will use the `INFO` log level, so make sure that `Console Log Level` is set to `INFO` in the `System Console`."
```

It's likely that the `File Log Level` is more commonly necessary to set in production. I'm not sure though. @justinegeffen Do you have any thoughts on this? We're telling an admin to make sure their Log Level is set to `INFO`, but it could be that they need to set `File Log Level` or `Console Log Level` based on how they plan on viewing their logs.

```suggestion
        "help_text": "Logging requests sent to S4B server for debugging purposes. Used only with Skype for Business Server. These logs will use the `INFO` log level, so make sure that your Log Level is set to `INFO` in the `System Console`."
```

---

Nice suggestion! This is a definitely better description for this option :+1: